instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/__init__.py

```diff
@@ -434,6 +434,8 @@ def __init__(self, filter_=None):
             else:
                 filter_ = any_of(filter_, excluded_service_filter)
         self._filter = filter_
+        self._request_hook = None
```

Comment:
Since it applies only to the client, perhaps the hooks should be called "client_request_hook", etc.